.github/workflows/build-scripts.yml

```diff
@@ -19,25 +19,8 @@ jobs:
       with:
         m2-cache-key: 'build-scripts'
 
-    - name: Run metabuild-common build script tests
-      run: clojure -M:test
-      working-directory: bin/common
-      timeout-minutes: 15
-    - name: Run build-drivers build script tests
-      run: clojure -M:test
-      working-directory: bin/build-drivers
-      timeout-minutes: 15
-    - name: Run i18n script tests
-      run: clojure -M:test
-      working-directory: bin/i18n
-      timeout-minutes: 15
-    - name: Run build-mb build script tests
-      run: clojure -M:test
-      working-directory: bin/build-mb
-      timeout-minutes: 15
-    - name: Run release script tests
-      run: clojure -M:test
-      working-directory: bin/release
+    - name: Run build and release script tests
+      run: clojure -X:dev:drivers:build:build-dev:build-test
```

Comment:
I should also mention that we can run all the build/release script tests in one step instead of 5 separate steps now. Well we could have done this all along probably but it's much easier now. So this will be nice and speed up CI a bit. Not like this was super slow in the first place, but this will be a nice benefit